src/views/RequiredSignoffs/ListSignoffs/index.jsx

```diff
@@ -58,28 +75,37 @@ function ListSignoffs() {
       {requiredSignoffs && (
         <Fragment>
           <div className={classes.toolbar}>
-            <Typography gutterBottom variant="h5">
-              Changes to Permissions
-            </Typography>
-            <TextField
-              className={classes.dropdown}
-              select
-              label="Product"
-              value={product}
-              onChange={handleFilterChange}>
-              {Object.keys(requiredSignoffs).map(product => (
-                <MenuItem key={product} value={product}>
-                  {product}
-                </MenuItem>
-              ))}
-            </TextField>
+            {permissionChanges && (
+              <Typography gutterBottom variant="h5">
+                Changes to Permissions
+              </Typography>
+            )}
+            {!permissionChanges && rulesOrReleasesChanges && (
+              <Typography gutterBottom variant="h5">
+                Changes to Rules / Releases
+              </Typography>
```

Comment:
This will make sure the title is still aligned horizontally with the dropdown when the permissions section is hidden.